docs/client/overview.rst

```diff
@@ -91,116 +88,145 @@ To select a particular spider to work with::
-``jobs`` field to work with the spider's jobs.
-
-To schedule a spider run::
+With ``.get``, you get a :class:`~scrapinghub.client.spiders.Spider`
+instance back.
 
-    >>> spider.jobs.run(job_args={'arg1': 'val1'})
-    <scrapinghub.client.Job at 0x106ee12e8>>
+.. note::
+    ``.list()`` does not return :class:`~scrapinghub.client.spiders.Spider`
+    instances. The ``id`` key in the returned dicts corresponds to
+    the ``.name`` attribute of :class:`~scrapinghub.client.spiders.Spider`
+    that you get with ``.get(<spidername>)``.
 
-Note that you don't need to specify spider name explicitly.
 
 .. _jobs:
 
-Jobs
-----
+Working with jobs collections
+-----------------------------
 
-:class:`~scrapinghub.client.jobs.Jobs` collection is available on project/spider
-level.
+Essentially, the purpose of spiders is to be run in Scrapinghub's platform.
+And a collection of jobs is represented by a :class:`~scrapinghub.client.jobs.Jobs`
+object.
 
-get
-^^^
-
-To select a specific job for a project::
-
-    >>> job = project.jobs.get('123/1/2')
-    >>> job.key
-    '123/1/2'
+Both project-level jobs (i.e. all jobs from a project) and spider-level jobs
+(i.e. all jobs for a specific spider) are available as a :class:`jobs
+<~scrapinghub.client.jobs.Jobs>` attribute of a
+:class:`~scrapinghub.client.projects.Project` instance
+or a :class:`~scrapinghub.client.spiders.Spider` instance respectively.
 
-Also there's a shortcut to get same job with client instance::
+Running jobs
+^^^^^^^^^^^^
 
-    >>> job = client.get_job('123/1/2')
+Use the ``run`` method to run a new job for a project or a particular spider::
 
-run
-^^^
+    >>> job = spider.jobs.run()
 
-Use ``run`` method to run a new job for project/spider::
+You can also use ``run`` at the project level, the difference being that
+a spider name is required::
 
-    >>> job = spider.jobs.run()
+    >>> job = project.jobs.run('spider1')
 
-Scheduling logic supports different options, like
+Scheduling jobs supports different options, like:
 
 - **job_args** to provide arguments for the job
 - **units** to specify amount of units to run the job
 - **job_settings** to pass additional settings for the job
-- **priority** to set higher/lower priority of the job
+- **priority** to set higher/lower priority for the job
 - **add_tag** to create a job with a set of initial tags
 - **meta** to pass additional custom metadata
 
-For example, to run a new job for a given spider with custom params::
+For example, to run a new job for a given spider with custom parameters::
 
     >>> job = spider.jobs.run(units=2, job_settings={'SETTING': 'VALUE'}, priority=1,
     ...                       add_tag=['tagA','tagB'], meta={'custom-data': 'val1'})
 
-Note that if you run a job on project level, spider name is required::
 
-    >>> job = project.jobs.run('spider1')
+Geting job information
+^^^^^^^^^^^^^^^^^^^^^^
 
-count
-^^^^^
+To select a specific job for a project, use ``.jobs.get(<jobID>)``::
```

Comment:
I'd replace `jobID` with `jobKey` or `JobKey`: my worries that it can be mixed with job id that is a part of job key, based on the concept that jobkey consists of 1) project id, 2) spider id and 3) job id.